apps/openmw/mwsound/soundmanagerimp.cpp

```diff
@@ -417,8 +418,20 @@ namespace MWSound
         else
             filelist = mMusicFiles[mCurrentPlaylist];
 
+        // If there are no tracks and the other playlist is populated, switch to the other playlist if it exists
         if(filelist.empty())
-            return;
+        {
+            if(mCurrentPlaylist == "Combat" && tracklist.find("Explore") != tracklist.end() && !tracklist["Explore"].empty())
```

Comment:
tracklist is a std::vector<int>, so you can not search for string. Also std::vector does not have the find() function. Do you mean the std::find?